packages/terra-paginator/src/Paginator.jsx

```diff
@@ -120,22 +119,18 @@ class Paginator extends React.Component {
     return pageButtons;
   }
 
-  hasNavContext() {
-    return this.props.totalCount && this.props.itemCountPerPage;
-  }
-
   defaultPaginator() {
     const theme = this.context;
     const { intl } = this.props;
-    const totalPages = calculatePages(this.props.totalCount, this.props.itemCountPerPage);
+    const totalPages = (this.props.totalCount) ? calculatePages(this.props.totalCount, this.props.itemCountPerPage) : 0;
```

Comment:
Can we update calculatePages to handle this logic?

Reply:
yes updated [here](https://github.com/cerner/terra-core/pull/3190/files/38eacfaf12b814588083bf00f69c9f88bbe7c405#diff-613eb3e1ac925d9653108c7638d0a3ea3c19e3601c2f22ca2299263e53c80510)